pipeline/frontend/yaml/compiler/compiler.go

```diff
@@ -108,17 +110,19 @@ func (c *Compiler) Compile(conf *yaml.Config) *backend.Config {
 	// add default clone step
 	if !c.local && len(conf.Clone.Containers) == 0 && !conf.SkipClone {
 		container := &yaml.Container{
-			Name:  "clone",
-			Image: "woodpeckerci/plugin-git:latest",
-			Vargs: map[string]interface{}{"depth": "0"},
-		}
-		// TODO: migrate to woodpeckerci/plugin-git:latest (multi arch)
-		switch c.metadata.Sys.Arch {
-		case "linux/arm":
-			container.Image = "plugins/git:linux-arm"
-		case "linux/arm64":
-			container.Image = "plugins/git:linux-arm64"
+			Name: "clone",
+			// Image:       "woodpeckerci/plugin-git:latest",
```

Comment:
Should we go with `:next` and add a todo and change it just before releasing 0.15.0?